### PR TITLE
Localization for Language of Simplified Chinese (zh-cn)

### PR DESCRIPTION
**Pull request by Li-Yanzhi**

### CODE REVIEW

File: hockeysdk/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <string name="hockeyapp_download_failed_dialog_positive_button">重试</string>
+
+    <!-- Update -->
+    <string name="hockeyapp_update_mandatory_toast">%s 有更新可用并且为强制更新! </string>
+    <string name="hockeyapp_update_dialog_title">有更新</string>
+    <string name="hockeyapp_update_button">更用</string>
+    <string name="hockeyapp_update_dialog_message">显示关于更新的信息？</string>
+    <string name="hockeyapp_update_dialog_negative_button">忽略</string>
+    <string name="hockeyapp_update_dialog_positive_button">显示</string>
+    <string name="hockeyapp_update_version_details_label" translatable="false">%1$s\n%2$s - %3$s</string>
```
Comment (ranterle) [inline hockeysdk/src/main/res/values-zh/strings.xml]: Please remove this, otherwise it breaks the build.

**Comment by ranterle** [comment]
Hi @Li-Yanzhi - thank you very much for providing this localization. Please do fix the following issues mentioned and we will make sure to review the changes for merging accordingly.

Best,
Mat

**Comment by Li-Yanzhi** [comment]
Done.

**Comment by ranterle** [comment]
Thanks @Li-Yanzhi for updating. And thanks for spotting the mismatch in some of our feedback strings. 
Please use a separate PR for fixing issues with another localization so we can merge them atomically and independently.
It would also be great if you could squash your remaining commits into a single one.

Thanks,
Mat

**Pull request closed without merging**